include/openssl/pem2.h

```diff
@@ -12,7 +12,6 @@ extern "C" {
 #endif
 
 #ifndef HEADER_PEM_H
-int ERR_load_PEM_strings(void);
 #endif
```

Comment:
Please remove the guard as well

Reply:
Hm. Additionally, should this file begin including pemerr.h?

Reply:
(done)

Reply:
Nope.  On second thought, this file should be removed entirely, and additionally, `crypto/err/err_all.c` should be changed to include the `<openssl/*err.h>` headers instead of the main ones...

Reply:
... and you can remove the inclusion of `<openssl/pem2.h>` in `include/openssl/pem.h`...

Reply:
...

come to think of it, you're right, we have to have this file include `<openssl/pemerr.h>` for the sake of the occasional poor guy who does include this one directly (should never happen, but it *is* a public header file, so...).

What I said about `crypto/err/err_all.c` and `include/openssl/pem.h` stands, though...

Reply:
I've tried to take this into account with two new commits just pushed.